WhatsNewInPython30.md

```diff
@@ -1,22 +1,22 @@
 Views And Iterators Instead Of Lists
 =======================
-- [ ] dict methods dict.keys(), dict.items() and dict.values() return �views� instead of lists. For example, this no longer works: k = d.keys(); k.sort(). Use k = sorted(d) instead (this works in Python 2.5 too and is just as efficient).
+- [ ] dict methods dict.keys(), dict.items() and dict.values() return �views� instead of lists. For example, this no longer works: k = d.keys(); k.sort(). Use k = sorted(d) instead (this works in Python 2.5 too and is just as efficient).
```

Comment:
Yeah, vs code messed them up. I willerge this one first and then merge the other to fix it up